xeth/xeth.go

```diff
@@ -370,71 +429,90 @@ func (self *XEth) RegisterFilter(earliest, latest int64, skip, max int, address
 	filter.SetAddress(cAddress(address))
 	filter.SetTopics(cTopics(topics))
 	filter.LogsCallback = func(logs state.Logs) {
-		self.logMut.Lock()
-		defer self.logMut.Unlock()
+		self.logMu.Lock()
+		defer self.logMu.Unlock()
 
-		self.logs[id].add(logs...)
+		self.logQueue[id].add(logs...)
 	}
 	id = self.filterManager.InstallFilter(filter)
-	self.logs[id] = &logFilter{timeout: time.Now()}
+	self.logQueue[id] = &logQueue{timeout: time.Now()}
 
 	return id
 }
 
-func (self *XEth) UninstallFilter(id int) bool {
-	if _, ok := self.logs[id]; ok {
-		delete(self.logs, id)
-		self.filterManager.UninstallFilter(id)
-		return true
-	}
+func (self *XEth) NewTransactionFilter() int {
```

Comment:
That's not the line you're looking for. But indeed, I implemented it as `eth_newTransactionFilter`. I'll change it to `eth_newPendingTransactionFilter`